src/derivatives/atlas.md

```diff
@@ -0,0 +1,961 @@
+      "atlas-Schaefer2018_seg-17n_scale-100_dseg.tsv": "",
+      "atlas-Schaefer2018_seg-17n_scale-200_dseg.tsv": "",
+      "atlas-Schaefer2018_seg-17n_scale-300_dseg.tsv": "",
+      "atlas-Schaefer2018_seg-kong17n_scale-100_dseg.tsv": "",
+      "atlas-Schaefer2018_seg-kong17n_scale-200_dseg.tsv": "",
+      "atlas-Schaefer2018_seg-kong17n_scale-300_dseg.tsv": "",
+      "sub-01": {
+         "anat": {
+            "sub-01_hemi-L_atlas-Schaefer2018_seg-7n_scale-100_den-164k_dseg.label.gii": "",
+            "sub-01_hemi-L_atlas-Schaefer2018_seg-7n_scale-200_den-164k_dseg.label.gii": "",
+            "sub-01_hemi-L_atlas-Schaefer2018_seg-7n_scale-300_den-164k_dseg.label.gii": "",
+            "sub-01_hemi-L_atlas-Schaefer2018_seg-17n_scale-100_den-164k_dseg.label.gii": "",
+            "sub-01_hemi-L_atlas-Schaefer2018_seg-17n_scale-200_den-164k_dseg.label.gii": "",
+            "sub-01_hemi-L_atlas-Schaefer2018_seg-17n_scale-300_den-164k_dseg.label.gii": "",
+            "sub-01_hemi-L_atlas-Schaefer2018_seg-kong17n_scale-100_den-164k_dseg.label.gii": "",
+            "sub-01_hemi-L_atlas-Schaefer2018_seg-kong17n_scale-200_den-164k_dseg.label.gii": "",
+            "sub-01_hemi-L_atlas-Schaefer2018_seg-kong17n_scale-300_den-164k_dseg.label.gii": "",
+            "...": "",
+            "sub-01_hemi-R_atlas-Schaefer2018_seg-kong17n_scale-300_den-164k_dseg.label.gii": "",
+         },
+         "bold": {
+            "sub-01_task-rest_hemi-L_den-164k_bold.func.gii": "",
+            "sub-01_task-rest_hemi-R_den-164k_bold.func.gii": "",
+         }
+      },
+   }
+})
+}}
+
+## Tabular data
+
+The `[probseg|dseg|mask|channels].tsv` file indexes and labels each node/parcel/region within the atlas.
+This file resembles the typical Look Up Table (LUT) often shared with atlases.
+This file will be essential for downstream workflows that generate matrices or other derived files within which node/parcel/region information is required,
+as the index/label fields will be used to reference the original anatomy the index/labels are derived from.
+Additional fields can be added with their respective definition/description in the sidecar json file.
+
+<table>
+  <tr>
+   <td>index (or placeholder in fragment in reference)
+   </td>
+   <td>REQUIRED. (Integer) The number associated with the node/parcel/region (right/left hemispheres may be different).
+   </td>
+  </tr>
+  <tr>
+   <td>label
+   </td>
+   <td>RECOMMENDED. The node name
+   </td>
+  </tr>
+  <tr>
+   <td>network_id
+   </td>
+   <td>OPTIONAL. Network ID the node/parcel belongs to
+   </td>
+  </tr>
+  <tr>
+   <td>network_label
+   </td>
+   <td>OPTIONAL. Label of Network (for example, Dorsal Attention Network)
+   </td>
+  </tr>
+  <tr>
+   <td>coordinate_report_strategy
+   </td>
+   <td>OPTIONAL (RECOMMENDED if x, y, z keys are specified).  The strategy used to assess and report x, y and z coordinates of a given node/parcel/region. For example, "CenterOfMass".
+   </td>
```

Comment:
This column should be removed because it's redundant with the `_<probseg|dseg|mask>.json` `CoordinateReportStrategy`.